XenModel/BrandManager.cs

```diff
@@ -31,6 +31,7 @@
 using System.Diagnostics;
 using System.Reflection;
 using System.Resources;
+using System.Web.UI;

```

Comment:
The changes in BrandManager.cs and AssemblyInfo.cs are not needed since we're not introducing any new branded values. If I remember well we said we would be adding two new entries `source_url` and `source_checksum` in the XCupdates.xml?

Reply:
source_url yes but i tought we werent bothered with the checksum on the source as it is just the source. We can though